### DISCUSSION TITLE
How to contact a repository owner?

### DISCUSSION THREAD

**Discussion by lbdyck** [community/community#21079] [Other Feature Feedback, Questions, & Ideas]
I have a question for a repository owner but I can find no way to submit a question to them.

That would be a very nice option.

**Comment by MrSarno** [comment]
Does the repository allow you to open an issue?

If not, their profile page may include a Twitter handle, e-mail address, or a link to a website with a contact form or something.

**Reply by lbdyck** [reply]
sadly no option for an issue and no e-mail or website mentioned. 

If GitHub allowed the ability to contact the owner via a private message that would be great - the owner can always use the delete key :)
